libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFAuthenticationManager.m

```diff
@@ -811,31 +768,19 @@ - (void)loginWithUser:(SFUserAccount*)account {
     [self.coordinator authenticate];
 }
 
-- (void)setupWithUser:(SFUserAccount*)account {
-    // sets the domain if it not set already
-    if (nil == account.credentials.domain) {
-        account.credentials.domain = [[SFUserAccountManager sharedInstance] loginHost];
-    }
-    
-    // if the user doesn't specify any scopes, let's use the ones
-    // defined in this account manager
-    if (nil == account.accessScopes) {
```

Comment:
Oh I see where the previous oversight on scopes in test came from—forgot that we actually used to use user-based scopes.  👍 for getting rid of those, as that's really app-level configuration.